game.py

```diff
@@ -1,9 +1,12 @@
 import random as rand
-from termcolor import colored, cprint
-from messages import random_rest_messages, random_mine_messages, random_fish_messages, random_combat_messages
+import platform
```

Comment:
needed this to get the platform. native function provided more information than was necessary and i didn't feel like splitting the results up.